src/sites/twitch-twilight/modules/player.jsx

```diff
@@ -758,7 +758,7 @@ export default class Player extends Module {
 
 
 	installVisibilityHook() {
```

Comment:
Does Safari natively consider the document as visible if a video is in picture-in-picture mode? If not, you have to adjust the property here to avoid Twitch dropping the video quality after one minute.

Reply:
If you do need to check for picture-in-picture mode for Safari, please remember that there may be multiple players with multiple video elements and you need to check them all from what I've seen here.